app/views/help/sign_in.html.erb

```diff
@@ -0,0 +1,96 @@
+<% content_for :title, "#{t('help.sign_in.title')} - GOV.UK" %>
+<%= render "govuk_publishing_components/components/breadcrumbs", {
+  breadcrumbs: [
+    {
+      title: "Home",
+      url: "/"
+    }
+  ]
+} %>
+
+<main id="content">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/title", title: t('help.sign_in.title') %>
+
+      <p class="govuk-body"><%= t('help.sign_in.intro') %></p>
+
+      <%= render "govuk_publishing_components/components/inset_text", {
+        text: t('help.sign_in.inset_text')
+      } %>
+
+      <%= tag.div class: "govuk-!-margin-bottom-9 govuk-!-margin-top-7" do %>
+        <p class="govuk-body govuk-!-margin-bottom-6"><%= t('help.sign_in.popular_services_intro') %></p>
```

Comment:
I think we'll want these bits in the published content item too, so that they show up in search.